composer/loggers/mosaicml_logger.py

```diff
@@ -167,7 +167,7 @@ def format_data_to_json_serializable(data: Any):
 
         # Unknown format catch-all
         return str(data)
-    except RuntimeError as e:
+    except Exception as e:
```

Comment:
what error were you getting in her? shouldn't the code in this part be pretty clear about the errors it raises? its just json serialization.